shared/enums.proto

```diff
@@ -67,6 +67,11 @@ enum EventType {
     SPECTRAL_LINE_RESPONSE = 68;
     CONCAT_STOKES_FILES = 69;
     CONCAT_STOKES_FILES_ACK = 70;
+    STOP_FILE_LIST = 71;
+    STOP_CATALOG_LIST = 72;
+    FILE_LIST_PROGRESS = 73;
```

Comment:
I don't see why we need three separate progress messages that effectively do the same thing. Just make a `FILE_LIST_PROGRESS` event with a `FileListProgress` message which contains a file type parameter.